manifests/config.pp

```diff
@@ -45,28 +45,12 @@
 
   $service_provider_lookup = pick(getvar('service_provider'), false)
 
-  if $service_provider_lookup != 'systemd' {
-    case $::osfamily {
-      'Debian': {
-        if $::lsbdistcodename == 'wheezy' {
-          $var_run_redis_mode  = '2755'
-          $var_run_redis_group = 'redis'
-        } else {
-          $var_run_redis_group = $::redis::config_group
-          $var_run_redis_mode = '2775'
-        }
-      }
-      default: {
-        $var_run_redis_mode = '0755'
-        $var_run_redis_group = $::redis::config_group
-      }
-    }
-
+  unless $facts['osfamily'] == 'Debian' or $service_provider_lookup == 'systemd' {
     file { '/var/run/redis':
       ensure => 'directory',
       owner  => $::redis::config_owner,
-      group  => $var_run_redis_group,
-      mode   => $var_run_redis_mode,
+      group  => $::redis::config_group,
```

Comment:
latest modulesync doesn't allow `::` after `$`, so you might want to directly remove it.

Reply:
IMHO not. The module is filled with it and this change keeps it consistent.

Reply:
That's only actually for classes.  I think there'll be a separate PR where the variables are sorted out.